bin/release-to-branch.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -e
+
+CURRENT_BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
+BRANCH_NAME="pre-release-$CURRENT_BRANCH_NAME"
+
+# Remove local branch is it already exists
+if [ `git branch --list $BRANCH_NAME` ]; then
+   echo "Deleting branch $BRANCH_NAME that already exists."
+   git branch -D $BRANCH_NAME
+   git push origin --delete $BRANCH_NAME
+fi
+
+# Build the package as normal
+npm run build:package
+
+git add package/
+
+git commit -m "Release GOV.UK Frontend to Github branch testing"
+
+# Create a local branch containing the package directory
+echo "Split package/ directory commits into `$BRANCH_NAME` branch..."
+git subtree split --prefix package -b $BRANCH_NAME
+
+# Force the push of the branch to the remote Github origin
+git push origin $BRANCH_NAME:$BRANCH_NAME
+
+# Remove build commit to bring the original branch back to where it was pre-release
+git reset HEAD^ --hard
+
+echo "To install this branch release run 'npm install --save alphagov/govuk-frontend#$BRANCH_NAME'"
```

Comment:
Should we add a message to make it clear that if you start making changes on the `prerelease-` branch you're possibly gonna get yourself in a bit of a mess?